test/fixtures/plugin.colors/pie.js

```diff
@@ -0,0 +1,23 @@
+module.exports = {
+  config: {
+    type: 'pie',
+    data: {
+      datasets: [
+        {
+          data: [0, 2, 4, null, 6, 8]
+        },
+        {
+          data: [5, 1, 6, 2, null, 9]
+        }
+      ]
+    },
+    options: {
+      plugins: {
+        legend: false,
+        colors: {
+          enabled: true
+        }
```

Comment:
nit: this is enabled by default right, so it is not nececarry to specify?

Reply:
yes but I used the same test source for the doughnut (changed only type).

Reply:
aah, yes makes sense, otherwise it would need an update to possible a lot of fictures 👍🏻